rpc/block_version.c

```diff
@@ -45,13 +45,19 @@ glusterBlockBuildMinCaps(void *data, operations opt)
     if (cblk->rb_size) {
       minCaps[GB_CREATE_RING_BUFFER_CAP] = true;
     }
+    if (cblk->io_timeout) {
+      minCaps[GB_CREATE_RING_BUFFER_CAP] = true;
+    }
```

Comment:
LOL, What is this @lxbsz  :-)

Reply:
It was introduced by the rebase diff conflicting.

Fixed them all.